endpoints/orders.yaml

```diff
@@ -320,7 +320,7 @@ paths:
         operationId: Enumerating subscriptions
         responses:
           '200':
-            description: Successful.
+            description: Your request was received and an asynchronous response payload will be delivered to the enumerating subscriptions counterparty endpoint. Counterparty endpoint URLs use the following endpoint to set URLs - /v1/orders/subscriptions.
```

Comment:
This the endpoint to view the currently set callback URLs, it's synchronous unlike the TVB flow endpoints. Update this description to something like this: "Response contains currently configured callback URLs"